documentation/vscode_and_bolt.md

```diff
@@ -0,0 +1,73 @@
+# Setting up VS Code for Bolt
+
+Bolt [configuration](configuring_bolt.md) and [inventory](inventory_file_v2.md)
+files are written in YAML that conforms to a particular specification. It also
```

Comment:
```suggestion
files are written in YAML that conforms to a particular specification.
```
Or just leave it in as a cliffhanger ;)

Reply:
It's basically a calling card for the team at this point. "It's critical that", "It also", ... :D